CONTRIBUTORS

```diff
@@ -106,3 +106,4 @@ worktycho
 Xenoxis
 xoft (Mattes Dolak/madmaxoft on GH)
 Yeeeeezus (Donated AlchemistVillage prefabs)
+CoolPuppyKid
```

Comment:
This file is alpha-sorted, you need to slot yourself in at the right alphabetical position!

Reply:
ok will do

Reply:
> This file is alpha-sorted, you need to slot yourself in at the right alphabetical position!

fixed now